bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts

```diff
@@ -280,11 +320,11 @@ export class KtbSequenceViewComponent implements OnInit, OnDestroy {
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  filtersChanged(event: DtQuickFilterChangeEvent<any> | { filters: any[] }): void {
+  filtersClicked(event: DtQuickFilterChangeEvent<any> | { filters: any[] }): void {
     this._seqFilters = event.filters as FilterType[];
-    this.sequenceFilters = this._seqFilters.reduce(
+    const sequenceFilters: Record<string, string[]> = this._seqFilters.reduce(
       (
-        filters: { [key: string]: string[] },
+        filters: Record<string, string[]>,
         currentFilter: [
```

Comment:
This is actually the mapping from `FilterType` which is `{ name: string; value: string }[]` to `Record<string, string[]>` which is `{ [key: string]: string[] }` so it can be added to the query params.